src/test/java/io/lettuce/core/pubsub/PubSubReactiveTest.java

```diff
@@ -251,6 +252,18 @@ void pubsubNumsub() {
         assertThat(result).containsKeys(channel);
     }
 
+    @Test
```

Comment:
Let's guard the test with `@EnabledOnCommand("SPUBLISH")` to ensure that the test only runs when Redis has the shared pub/sub functionality. We typically keep the tests guarded to run tests against lower Redis versions if necessary.

Reply:
Makes sense, will do